.mergify.yml

```diff
@@ -183,6 +183,24 @@ pull_request_rules:
           2. Open a follow-up PR to update the JavaScript client `@solana/web3.js` ([example](https://github.com/solana-labs/solana-web3.js/pull/2868/files))
 
           Thank you for keeping the RPC clients in sync with the server API @{{author}}.
+  - name: Reminder to add Firedancer team to changes in `programs/`
+    conditions:
+      - or:
+        - files~=^programs/address-lookup-table/src/.*\.rs$
+        - files~=^programs/bpf_loader/src/.*\.rs$
+        - files~=^programs/compute_budget/src/.*\.rs$
+        - files~=^programs/config/src/.*\.rs$
+        - files~=^programs/loader-v4/src/.*\.rs$
+        - files~=^programs/stake/src/.*\.rs$
+        - files~=^programs/system/src/.*\.rs$
+        - files~=^programs/vote/src/.*\.rs$
+        - files~=^programs/zk-elgamal-proof/src/.*\.rs$
+    actions:
+      comment:
+        message: |
+          If this PR represents a change to a native program implementation, even
+          a refactor, please include someone from the Firedancer team as one of the
+          reviewers.
```

Comment:
Could we change this to encourage developers to keep changes to the native code to a minimum? The wording in the PR description was pretty good. We should also make it clear that this request is limited to just implementation, not tests.

Reply:
Updated the language in [1e2d298](https://github.com/anza-xyz/agave/pull/2233/commits/1e2d29819bfdcfb2f5e895013b9e961c7c11ab98)